low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -309,10 +305,12 @@ def validate(self):
     def process(self):
         #self.validate_consul_config.validate_config()
         if os.path.exists(self.SSPL_CONFIGURED):
-            sys.exit(0)
+            return
         syslog.openlog(logoption=syslog.LOG_PID, facility=syslog.LOG_LOCAL3)
         syslog.syslog(syslog.LOG_ERR, f"SSPL is not configured. Run provisioner scripts in {self._script_dir}.")
-        sys.exit(1)
+        raise SetupError(errno.EINVAL,
+                "SSPL is not configured. Run provisioner scripts in %s.",
```

Comment:
this statement may not apply, as sspl_setup can be triggered by sspl-dev_deploy or cortx provisioner eventually. @mariyappanp please take care of this